styleguide/derek/_partials/_basement.ejs

```diff
@@ -1,17 +1,15 @@
 <!--[basement]-->
 <div class="bg-dark-gray">
-  <div class="container basement">
+  <div class="basement">
     <div class="row">
```

Comment:
lets change `row` class to `basement-row` and then use the `make-row()` mixin in css.